Banking/src/main/java/com/example/Banking/model/UserTransaction.java

```diff
@@ -0,0 +1,29 @@
+package com.example.Banking.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "transaction")
+public class UserTransaction {
+
```

Comment:
We also need a status column. A transaction could be success,failed,pending. Pending when transaction started, then updated to success or failed.

Reply:
okay, will update now